test/cmd/images.sh

```diff
@@ -72,7 +72,7 @@ os::cmd::expect_success_and_text 'oc describe istag/mysql:latest' 'Image Created
 os::cmd::expect_success_and_text 'oc describe istag/mysql:latest' 'Image Name:'
 name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
 imagename="isimage/mysql@${name:0:15}"
-os::cmd::expect_success 'oc describe "${imagename}"'
+os::cmd::expect_success "oc describe ${imagename}"
```

Comment:
> In the README I suggest always double-quoting in the test file so that the output to the command line ("Running ..." or "SUCCESS:", etc) has the variables and you can debug better.

We can start there, but I'm actually not sure we'll always want that.  We want stable names to track historical flakes.  If the names change because the command has resources that have generated names, then the test name would continually change.